pdsClient/pdsClient.go

```diff
@@ -748,6 +748,18 @@ func (c *E3dbPDSClient) GetOrCreateAccessKey(ctx context.Context, params GetOrCr
 	return e3dbClients.DecryptEAK(encryptedAccessKey, accessKeyResponse.AuthorizerPublicKey.Curve25519, rawEncryptionKey)
 }
 
+// GetFileRecord gets a record for the file with the specified pendingFileID and returns it.
+func (c *E3dbPDSClient) GetFileRecord(ctx context.Context, pendingFileID uuid.UUID) (*Record, error) {
```

Comment:
`GetPendingFileRecord` please

Reply:
This was a mistake with variable names -- it should be fileID instead of pendingFileID since the file has already been committed. I've updated the names.